src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,227 @@
+package duke.command;
```

Comment:
maybe can try to split command into different class for more OOP

Reply:
Sure! Thanks for the suggestions!